src/librustc_resolve/lib.rs

```diff
@@ -1359,6 +1394,7 @@ pub struct Resolver<'a> {
     injected_crate: Option<Module<'a>>,
 }
 
+/// Nothing really interesting here, it just provides memory for the rest of the crate.
```

Comment:
nit: memory -> arenas

Reply:
I think it is obvious it contains arenas, my point was that these arenas are there to allocate memory. Should I still change it?